.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
All the files which are connected to local development shouldn't be added to version control.
It should be ignored by .gitignore file
https://www.atlassian.com/git/tutorials/saving-changes/gitignore